geometry_msgs/Point32 P1
geometry_msgs/Point32 P2
geometry_msgs/Point32 P3
---
geometry_msgs/Point32 P0
geometry_msgs/Point32 n
